agent/util-scripts/pbench-remote-sysinfo-dump

```diff
@@ -28,7 +28,7 @@ fi
 
 cd $sysinfo_path >&2
 echo "pbench-sysinfo-dump data size: $(du -sm $sysinfo_path | awk '{print $1}') MB" >&2
-tar cf - *
+tar --xz -cf - *
```

Comment:
Let's revert this change, and instead, in `pbench-collect-sysinfo`, just drop the `.xz` suffix from the local file name.